src/utils/icon.ts

```diff
@@ -95,6 +96,13 @@ const lockedLogos: LogosByEnvironment = {
 export async function updateIcon(hasPerms: boolean) {
   const val = await ExtensionStorage.get("decryption_key");
 
+  if (import.meta.env?.VITE_IS_EMBEDDED_APP === "1") {
+    postEmbeddedMessage({
```

Comment:
Embed has no icon, but it has the connection indicator, which has similar states. There's no "locked" state though.